gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-bin.zip
```

Comment:
Why 5.0? It appears to be the version of Gradle that added support for Java 11. Maybe it could be higher but the build appears to run now at least (`checkstyle` notwithstanding) 🤷🏻‍♂️